packages/adapter-components/src/filters/referenced_instance_names.ts

```diff
@@ -63,162 +60,40 @@ const log = logger(module)
 const { isDefined } = lowerDashValues
 const { createReferencesTransformFunc } = references
 
-type TransformationIdConfig = {
-  idFields: string[]
-  extendsParentId?: boolean
-  nameMapping?: NameMappingOptions
-}
-
-const getFirstParentElemId = (instance: InstanceElement): ElemID | undefined => {
+const getFirstParent = (instance: InstanceElement): InstanceElement | undefined => {
   const parentsElemIds = getParents(instance)
     .filter(parent => isReferenceExpression(parent) && isInstanceElement(parent.value))
-    .map(parent => parent.elemID)
+    .map(parent => parent.value)
   // we are only using the first parent ElemId
```

Comment:
nit (since you switched to value)
```suggestion
  // we are only using the first parent
```